tests/cases/fourslash/indentationInAmdIife.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference path="fourslash.ts"/>
+
+//// function foo(a?,b?) { b(a); }
+//// 
+//// (foo)(1, function() {/*4_0*/
+//// });
+//// 
+//// // No line-breaks in the expression part of the call expression
+//// 
+//// (foo)
+////     (1, function () {/*8_0*/
+////     });
+//// (foo)(1,
+////     function () {/*8_1*/
+////     });
+//// (foo)(1, function() 
+//// {/*4_2*/
+//// });
+//// 
+//// // Contains line-breaks in the expression part of the call expression.
+////
+//// (
+////     foo)(1, function () {/*4_1*/
+////     });
+//// (foo
+//// )(1, function () {/*4_3*/
+//// });
+//// (foo
+//// )
+////     (1, function () {/*8_2*/
+////     });
+//// (foo
+//// )(1,
+////     function () {/*8_3*/
+////     });
+//// (foo
+//// )(1, function()
+//// {/*4_4*/
+//// });

```

Comment:
Do we have existing coverage for other areas that could regress (e.g. typing newlines inside other expressions, arrays, objects, argument lists, etc.)? We should make sure these don't regress. (i.e. if there is no existing coverage, we should add some basic tests with this change).

Reply:
tests added.